lib/utils/log/slog.go

```diff
@@ -34,8 +37,56 @@ const (
 
 	// TraceLevelText is the text representation of Trace verbosity.
 	TraceLevelText = "TRACE"
+
+	noColor = -1
```

Comment:
Is it worth keeping the added code in a file other than slog.go?

Are there parts of it we expect to delete alongside the last remains of logrus? Those could be in a separate file too (that could be deleted in its entirety).

Reply:
Sounds good.